docs/changelog.rst

```diff
@@ -1,6 +1,10 @@
 :desc: Rasa Core Changelog
 
 .. include:: ../CHANGELOG.rst
+Added
+-------
+- Added `priority` property of policies to influence best policy in the case of equal confidence
```

Comment:
need to add to migration that one needs to pass priority property to the super class call if user wrote their own policy

Reply:
only if they want to define a priority right? by mixing your suggestions I was thinking `policy.__init__()` can include `priority: Optional[int] = 1`

Reply:
yes, if we have a sane default prio, no migration is necessary